.github/workflows/CI.yml

```diff
@@ -44,24 +44,34 @@ jobs:
       if: matrix.python-version != 'pypy-3.7'
       run: |
         which python
+
     - name: Run tests with headless Chrome and with PyPy
       if: matrix.python-version == 'pypy-3.7'
       run: |
         xvfb-run --auto-servernum python atest/run.py --nounit --zip headlesschrome
-    - name: Run tests with headless Chrome and with Python != 3.8
-      if: matrix.python-version != '3.8' && matrix.python-version != 'pypy-3.7'
+
+    - name: Run tests with normal Chrome and with Python 3.7
+      if: matrix.python-version == '3.7'
```

Comment:
Good catch, I will fix this to normal chrome. 3.8 was not tested previously, despite what some of the task names state. This is because 3.8 was not(and is not) included in the matrix.

I would prefer to get rid of this `if python-version == xx && yy=zz` and just use the matrix to define browsers and python+rf versions. Maybe even Selenium versions in the future, though this is not realistic at the moment. I realize this results in a lot of jobs very quickly but as far as I can tell Github Actions are free for public repositories and the only limit is 20 concurrent jobs. I think this would further encourage making current tests more robust or not-flaky by fixing timing issues and such, and also to help create robust tests in the future.

Reply:
The conditions on python version I believe were to limit the overall matrix. I'm good with expanding it. I would say possibly in another pull request.

Reply:
Yeah definitely not doing that in this pull request.